pkg/vex/vex.go

```diff
@@ -21,38 +21,57 @@ import (
 )
 
 const (
-	// This is the format identifier for
+	// formatIdentifier is the identifier value used in a VEX document's Format field.
 	formatIdentifier = "text/vex+json"
 
-	// MIME type to record in the attestations
-	MimeType = "text/vex"
+	// TypeURI is the type used to describe VEX documents, e.g. within [in-toto
+	// statements].
+	//
+	// [in-toto statements]: https://github.com/in-toto/attestation/blob/main/spec/README.md#statement
+	TypeURI = "text/vex"
```

Comment:
@puerco I renamed this to map to the in-toto spec, since that's how we're using this value: they say it should be a Type URI, not a MIME type. This means we also need to come up with a valid URI — I left the old value (a MIME type) for now but we should talk about what the correct value would be.

Reply:
Makes sense, once we set up a home for the schema we can point to it to keep it synched with future versions.